create_sample_data.py

```diff
@@ -84,6 +93,12 @@ def check_dataset_lock(dataset_dbid):
                 }
                 resp = requests.post(base_url + '/api/files/' + str(file_id) + '/metadata', data=data, headers=headers, stream=True, files=buff)
                 print(resp)
+                # and finally, restrict the file if requested (in the same optional file metadata file, above):
+                if 'restricted' in file_metadata.keys():
+                    if (file_metadata['restricted']):
+                        headers['Content-Type'] = 'application/octet-stream'
+                        resp = requests.put(base_url + '/api/files/' + str(file_id) + '/restrict', data='true', headers=headers)
```

Comment:
It should be possible to restrict files on upload but if this is easier (restricting it with a separate API call), especially for now, that's just fine.

Reply:
I feel like a separate API call should be ok for now. Since we are making additional API calls to modify filemetadata anyway.